From 834e0dc0805433f8ea768aa4fb32bf1676078782 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Mon, 22 Feb 2021 15:14:58 +0200
Subject: [PATCH] [SW-20154] gaudi2: onix: remove the key location macro

Key location has now been changed to 2 bits instead of
a single bit. A value of 0 will always puts
the Keys default location to ROM. Hence no need of an
explicit macro.

Change-Id: Id4137104eb6cea25f579873bad8d650ee73605fc
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/onix.c b/gaudi2_arc_gnu/onix.c
index a3ce726..1d90e3c 100644
--- a/gaudi2_arc_gnu/onix.c
+++ b/gaudi2_arc_gnu/onix.c
@@ -11,7 +11,6 @@
 
 #define EFUSE_SYSCFG_ONIX_DEFAULT \
 				(EFUSE_SYSCFG_FW_VERSION_LOCATION_SPI | \
-				 EFUSE_SYSCFG_KEY_LOCATION_ROM | \
 				 EFUSE_SYSCFG_REVOKATION_LOCATION_SPI | \
 				 EFUSE_SYSCFG_ENABLE_VALIDATION_MASK | \
 				 EFUSE_SYSCFG_GPIO02_RECOVERY_CONTROL)
